p2p/discover/node.go

```diff
@@ -230,6 +230,10 @@ func (n NodeID) Bytes() []byte {
 	return n[:]
 }
 
+func (n NodeID) Address() []byte {
```

Comment:
I am pretty sure this is the wrong place to introduce an implementation to the `OverlayPeer` interface, but I couldn't find many types that actually implement it.

Reply:
you dont need this. the nodeID is not an overlay address but underlay

Reply:
I figured this is the wrong place. Will correct this.